src/translation.json

```diff
@@ -445,7 +445,7 @@
       "exitProgram": "Exit",
       "encounterDate": "Visit Date",
       "visitDetails": "Visit Details",
-      "registrationDate": "Date of Registration",
+      "registrationDate": "Registration date",
```

Comment:
Do not modify this file. Raise a PR as I pointed out in community gitter room. 
```
I have merged the new translations key to staging environment so you will no longer need to add them to json files on avni-webapp repo
And also switching the webapp to fetch it from api instead of files
And if you need to add any more new keys please add them directly to translation files in https://github.com/persistent-avni-contribution/avni-client and raise a pull request as usual
```

Reply:
@DarshanaTakPSL Delete this file as it's not used anymore.